protocol/x/rewards/types/params.go

```diff
@@ -10,9 +10,9 @@ import (
 func DefaultParams() Params {
 	return Params{
 		// Corresponds to module account address: dydx16wrau2x4tsg033xfrrdpae6kxfn9kyuerr5jjp
-		TreasuryAccount:  "rewards_treasury",
+		TreasuryAccount:  TreasuryAccountName,
 		Denom:            "dv4tnt",
```

Comment:
nit: don't we define `dv4tnt` somewhere as a constant? can we reuse that?

Reply:
We actually don't right now - prefer to keep as is to avoid conflict with this other [PR](https://github.com/dydxprotocol/v4-chain/pull/454)